guides/basic_auth.md

```diff
@@ -0,0 +1,51 @@
+### <u> Basic Authentication </u>
+
+You would require a valid **username** and **password** of an [Administrator](https://help.sap.com/docs/connectivity/sap-btp-connectivity-cf/cloud-connector-initial-configuration#loiodb9170a7d97610148537d5a84bf79ba2__log_in) to get authenticated.
+ 
+There are multiple ways to configure your credentials.
+
+1. You can configure them as part of the provider configuration as shown below:
+
+    ```hcl
```

Comment:
I would propose to use `terraform` instead of `hcl`. The later is a bit dated and should not be used any longer although it still works